contracts/adapters/KycOnboarding.sol

```diff
@@ -243,22 +264,28 @@ contract KycOnboardingContract is
         address kycedMember,
         address tokenAddr,
         uint256 amount,
+        uint256 memberNonce,
         bytes memory signature
     ) internal reimbursable(dao) {
         require(
-            !isActiveMember(dao, dao.getCurrentDelegateKey(kycedMember)),
+            !isActiveMember(dao, dao.getCurrentDelegateKey(kycedMember)) ||
+                _daoCanTopUp(dao, tokenAddr),
```

Comment:
In that case it's technically not a top up, but the first redemption and top ups share the same logic.
`!isActiveMember || _daoCanTopUp` and `!isActiveMember || (isActiveMember && _daoCanTopUp)` should resolve the same way. Do we need a distinction on whether an onboarding is a top up or not?

Reply:
Gotcha. `!isActiveMember || _daoCanTopUp` should be enough. The first time the `kycMember` is not active, so that will pass. Any operation after that, for the same `kycMember`, will be verified with the `_daoCanTopUp`. It is all good then. Regarding the operation type, I don't think need that distinction here at this stage, but it would be nice to double check with @adridadou. One idea would be to change the `Onboarded` event to support a `topUp` flag.

Reply:
I don't think that knowing whether it was a top up or not is important information. If we have previous events then it is a top up. 

The only thing I see here that we need to take into account is that the change in EIP-712 structure means that we need to change how the kyc attestation is created on the kyc backend side to add the nonce.